action/action.go

```diff
@@ -305,7 +300,10 @@ func AssembleSealedEnvelope(act Envelope, addr string, pk keypair.PublicKey, sig
 // Verify verifies the action using sender's public key
 func Verify(sealed SealedEnvelope) error {
 	hash := sealed.Hash()
-	if success := crypto.EC283.Verify(sealed.SrcPubkey(), hash[:], sealed.Signature()); success {
+	if len(sealed.Signature()) != 65 {
```

Comment:
where does this magic number 65 come from? please add the comment and make it a constant somewhere.

Reply:
Done.